lib/core/datatable/data/data-column.model.ts

```diff
@@ -44,4 +45,5 @@ export interface DataColumn {
     focus?: boolean;
     sortingKey?: string;
     header?: TemplateRef<any>;
+    draggable?: boolean;
```

Comment:
Here I am not sure if would be better to just enable drag and drop for specific column or just enable it for whole table at once, currently we can enable it for each column separately (@VitoAlbano if you think it is not necessary I can change it, should be quick change)